src/thread.js

```diff
@@ -47,9 +104,9 @@ class Thread {
     this._requireLoad()
     if (!opts.limit) opts.limit = -1
     return this._db.iterator(opts).collect().map(entry => {
-      let post = entry.payload.value
-      post.postId = entry.hash
-      return post
+      const post = entry.payload.value
+      const metaData = { postId: entry.hash, author: entry.identity.id }
```

Comment:
So given this a post would look like:
```
{
  message: <msg-object>,
  timestamp: <ts>,
  metaData: {
    postId: <hash>,
    author: <did>
  }
}
```

Seems weird to me to keep timestamp outside of the metadata. Also I would prefer `metadata` (without a capital D). The resulting data structure would look like: 
```
{
  message: <msg-object>,
  metadata: {
    timestamp: <ts>,
    postId: <hash>,
    author: <did>
  }
}
```

Reply:
its actually flat, similarly to before 
```
{
  message: <msg-object>,
  timestamp: <ts>,
  postId: <hash>,
  author: <did>
}
```

Reply:
Oh, sorry didn't read well enough!